theano/sandbox/cuda/basic_ops.py

```diff
@@ -3656,8 +3635,6 @@ class GpuAllocEmpty(GpuOp):
 
     """
 
-    __props__ = ()
```

Comment:
Keep it. When there is no init, it mean empty props. empty props isn't the same as no props.